src/components/Nav.astro

```diff
@@ -1,17 +1,53 @@
 <nav
-    class="max-w-screen-2xl w-10/12 mx-auto pt-10 animate__animated animate__fadeIn animate__delay-2s"
+	class='max-w-screen-2xl w-10/12 mx-auto pt-10 animate__animated animate__fadeIn animate__delay-2s'
```

Comment:
We should probably tackle #12 soon so we can clean up PRs a bit - no action needed here just noting!